src/go/rpk/pkg/cli/registry/schema/get.go

```diff
@@ -41,6 +42,8 @@ potential (mutually exclusive) ways:
 * By version, returning a schema for a required subject and version
 * By ID, returning all subjects using the schema, or filtering for one subject
 * By schema, checking if the schema has been created in the subject
+
+You may print the schema using the '--print-schema' flag.
 `,
 		Args: cobra.MaximumNArgs(1),
 		Run: func(cmd *cobra.Command, args []string) {
```

Comment:
f.Help two lines below this comment is no longer correct if the end-user uses --print-schema. Should we:
* do nothing, and print the raw schema as is in this PR
* conditionally change Help to print a different type, `{schema: <schema_text>}`
* change Help to always print a different type, `{usages_or_something: []subjectSchema, schema: <schema_text>}`
?

Reply:
I think conditionally changing the help text is better:

```
$ rpk registry schema get --format help
[]{
  subject: string
  version: int
  id: int
  type: string
}
$ rpk registry schema get --format help --print-schema 
string

```

Reply:
I like it 👍